vsintegration/src/FSharp.Editor/Hints/InlineParameterNameHints.fs

```diff
@@ -43,20 +42,19 @@ module InlineParameterNameHints =
         |> Option.filter (not << Seq.isEmpty)
         |> Option.defaultValue Seq.empty
 
-    let private getTupleRanges =
-        Seq.map (fun location -> location.ArgumentRange) >> Seq.toList
+    let getTupleRanges = Seq.map (fun location -> location.ArgumentRange) >> Seq.toList
```

Comment:
Any reason we are explicitly want list here as opposed to sequence?

Reply:
That's because curry ranges are natively a list for some reason - but yeah it wasn't really clear from the code and can be done other way round, updated.